static/app/views/dashboards/widgets/lineChartWidget/lineChartWidget.stories.tsx

```diff
@@ -83,6 +84,41 @@ export default storyBook(LineChartWidget, story => {
       </Fragment>
     );
   });
+
+  story('Colours', () => {
```

Comment:
```suggestion
  story('Colors', () => {
```

Only because the prop is named color :D